README.md

```diff
@@ -28,22 +29,42 @@ Add the following in your Synapse config under `account_validity`:
       renew_at: 1w
 ```
 
-Also under the HTTP client `listener`, configure an `additional_resource` as per below:
-
-```yaml
-    additional_resources:
-      "/_synapse/client/email_account_validity":
-        module: email_account_validity.EmailAccountValidityServlet
-        config:
-          # The maximum amount of time an account can stay valid for without being
-          # renewed.
-          period: 6w
-```
-
 The syntax for durations is the same as in the rest of Synapse's configuration file.
 
-If they are not already there, copy the [templates](/email_account_validity/templates)
-into Synapse's templates directory.
+## Templates
+
+The templates the module will use are:
+
+* `notice_expiry.(html|txt)`: The content of the renewal email. It gets passed the
+  following variables:
+    * `app_name`: The value configured for `app_name` in the Synapse configuration file
+      (under the `email` section).
+    * `display_name`: The display name of the user needing renewal.
+    * `expiration_ts`: A timestamp in milliseconds representing when the account will
+      expire. Templates can use the `format_ts` (with a date format as the function's
+      parameter) to format this timestamp into a human-readable date.
+    * `url`: The URL the user is supposed to click on to renew their account. If
+      `send_links` is set to `false` in the module's configuration, the value of this
+      variable will be `None`.
+    * `renewal_token`: The token to use in order to renew the user's account. If
+      `send_links` is set to `false`, templates should prefer this variable to `url`.
+* `account_renewed.html`: The HTML to display to a user when they successfully renew
+  their account. It gets passed the following vaiables:
+    * `expiration_ts`: A timestamp in milliseconds representing when the account will
+      expire. Templates can use the `format_ts` (with a date format as the function's
+      parameter) to format this timestamp into a human-readable date.
+* `account_previously_renewed.html`: The HTML to display to a user when they try to renew
+  their account with a token that's valid but previously used. It gets passed the same
+  variables as `account_renewed.html`.
+* `invalid_token.html`: The HTML to display to a user when they try to renew their account
+  with the wrong token. It doesn't get passed any variable.
+
+You can find and change the default templates [here](https://github.com/matrix-org/synapse-email-account-validity/tree/main/email_account_validity/templates).
+Admins can install custom templates either by changing the default ones directly, or by
+configuring Synapse with a custom template directory that contains the custom templates.
+Admins that don't need to customise their templates can just use the module as is and ignore this paragraph.
```

Comment:
ok, how does this work?

it looks like we call `module_api.read_templates` (with no custom template dir), which calls `config._base.read_templates`, which just looks in `synapse/res/templates`. How does it find templates in `email_account_validity/templates`?